.github/workflows/ci-tests.yml

```diff
@@ -74,7 +74,7 @@ jobs:
         pip install tox
     - name: Run the tests
       run: |
-        tox -e py -- -n 2 test
+        tox -e py -- --cov=sqlfluff -n 2 test
```

Comment:
For test runs that upload coverage checks, we want to specify **which** code we're targeting for coverage. Previously, we were always specifying `sqlfluff`, even for the dbt templater tests, which **should've** been targeting `sqlfluff_templater_dbt`.